services/block_service_test.go

```diff
@@ -40,9 +40,24 @@ import (
 )
 
 const (
-	hsh = "0xd83b1dcf7d47c4115d78ce0361587604e8157591b118bd64ada02e86c9d5ca7e"
+  hsh = "0xd83b1dcf7d47c4115d78ce0361587604e8157591b118bd64ada02e86c9d5ca7e"
 )
 
+// nolint:unused
+func loadTransfers(transferFile string) []*client.EVMTransfer {
```

Comment:
sorry left this function in the generic-sdk by mistake, but we don't need it or use it, so you can just get rid of it